Blank identifier causes panic

The following program `sample.go` triggers a panic:

```go
package main

func main() {
	_ = 1
}
```

Expected result:
```console
$ go run ./sample.go
```

Got:
```console
$ yaegi ./sample.go
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xa8 pc=0x60ddda]

goroutine 1 [running]:
github.com/containous/yaegi/interp.(*itype).TypeOf(0x0, 0x145535f, 0x1)
	/home/cuonglm/go/src/github.com/containous/yaegi/interp/type.go:676 +0x3a
github.com/containous/yaegi/interp.(*Interpreter).cfg.func2(0xc0001ca800)
	/home/cuonglm/go/src/github.com/containous/yaegi/interp/cfg.go:374 +0x5cfe
github.com/containous/yaegi/interp.(*node).Walk(0xc0001ca800, 0xc000193d68, 0xc000193da0)
	/home/cuonglm/go/src/github.com/containous/yaegi/interp/interp.go:113 +0xaf
github.com/containous/yaegi/interp.(*node).Walk(0xc0001ca700, 0xc000193d68, 0xc000193da0)
	/home/cuonglm/go/src/github.com/containous/yaegi/interp/interp.go:110 +0x70
github.com/containous/yaegi/interp.(*node).Walk(0xc0001ca200, 0xc000193d68, 0xc000193da0)
	/home/cuonglm/go/src/github.com/containous/yaegi/interp/interp.go:110 +0x70
github.com/containous/yaegi/interp.(*node).Walk(0xc0001ca000, 0xc000193d68, 0xc000193da0)
	/home/cuonglm/go/src/github.com/containous/yaegi/interp/interp.go:110 +0x70
github.com/containous/yaegi/interp.(*Interpreter).cfg(0xc0001bc000, 0xc0001ca000, 0xc000140080, 0x4, 0x0, 0x0, 0x4)
	/home/cuonglm/go/src/github.com/containous/yaegi/interp/cfg.go:39 +0x1b2
github.com/containous/yaegi/interp.(*Interpreter).Eval(0xc0001bc000, 0xc0001c4000, 0x25, 0x225, 0xc0001c4000, 0x25, 0x0, 0xc2f260)
	/home/cuonglm/go/src/github.com/containous/yaegi/interp/interp.go:257 +0x135
main.main()
	/home/cuonglm/go/src/github.com/containous/yaegi/cmd/yaegi/yaegi.go:77 +0x2e3
```

--- a/_test/assign8.go
+++ b/_test/assign8.go
@@ -1,0 +1,9 @@
+package main
+
+func main() {
+	_ = 1
+	println(1)
+}
+
+// Output:
+// 1

--- a/interp/cfg.go
+++ b/interp/cfg.go
@@ -343,7 +343,7 @@ func (interp *Interpreter) cfg(root *node) ([]*node, error) {
 				dest, src := n.child[i], n.child[sbase+i]
 				var sym *symbol
 				var level int
-				if n.kind == defineStmt {
+				if n.kind == defineStmt || (n.kind == assignStmt && dest.ident == "_") {
 					if src.typ != nil && src.typ.cat == nilT {
 						err = src.cfgErrorf("use of untyped nil")
 						break

